CMakeLists.txt

```diff
@@ -48,7 +52,10 @@ set(SDL12COMPAT_SRCS
 )
 add_library(SDL SHARED ${SDL12COMPAT_SRCS})
 
-include("cmake/modules/FindSDL2.cmake")
```

Comment:
I guess we can remove the module file itself after this?

Reply:
No, it's still used through `find_package(SDL2)`.
Using `find_package(SDL2)` allows using an installed `SDL2Config.cmake`.
We added some level of indirection here.